internal/coreconfig/namespace.go

```diff
@@ -0,0 +1,26 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package coreconfig
+
+const (
+	// NamespaceName is a short name for a pre-defined namespace
+	NamespaceName = "name"
+	// NamespaceName is a long description for a pre-defined namespace
+	NamespaceDescription = "description"
+	// NamespaceOrgKey is a default signing key for blockchain transactions within a namespace (overrides top-level org.key)
+	NamespaceOrgKey = "org.key"
```

Comment:
Per comments on the FIR, I think this should be in a `multiparty.rootIdentity` (or similar) scoped sub-section of the config, in the new world.

Reply:
Yea, still an [open discussion] (https://github.com/hyperledger/firefly-fir/pull/12#discussion_r870661545) on exactly how to spell that. I think it does need to have "org" in the name though, since the root identity is ultimately always an org.

Reply:
I guess the question is whether we want to merge this and go through a period of flux in `main`, or hold it out until we close on the config design.

Reply:
Seems like the discussion on the FIR has resolved - will incorporate those changes back to here.

Reply:
Seems like the discussion on the FIR has converged for now, so I've reworked the config here to match. Changes in the new commit:

- Replace "mode: multiparty" with "multiparty.enabled: true"
- Default to multiparty mode if org name or key is set
- Remove previously-deprecated "org.identity" root key
- Add new "defaultKey" config to namespaces (valid for both multiparty and gateway namespaces, overrides "multiparty.org.key" for signing)